torch/onnx/utils.py

```diff
@@ -485,6 +485,10 @@ def _export(model, args, f, export_params=True, verbose=False, training=False,
                 {}, opset_version, dynamic_axes, False, operator_export_type,
                 strip_doc_string, val_keep_init_as_ip, custom_opsets, val_add_node_names)
 
+        if operator_export_type != OperatorExportTypes.ONNX_ATEN_FALLBACK:
```

Comment:
Shall we add a parameter in torch.onnx.export to control this check? I am wondering if someone would like to export some special (also invalid) model, then we suddenly block them.

Reply:
yeah thats probably a good idea to be able to disable it if needed.